src/components/Offramp/Confirm.view.tsx

```diff
@@ -277,13 +286,13 @@ export const OfframpConfirmView = ({
         }
 
         const route = await utils.fetchRouteRaw(
-            claimLinkData.tokenAddress,
-            claimLinkData.chainId.toString(),
+            claimLinkData.tokenAddress!,
+            claimLinkData.chainId!.toString(),
             usdcAddressOptimism,
             optimismChainId,
-            claimLinkData.tokenDecimals,
-            claimLinkData.tokenAmount,
-            claimLinkData.senderAddress
+            claimLinkData.tokenDecimals!,
+            claimLinkData.tokenAmount!,
+            claimLinkData.senderAddress ?? '0x9647BB6a598c2675310c512e0566B60a5aEE6261'
```

Comment:
yep this is bad and even superfluous because i alrdy added it to utils fetch func:
![image](https://github.com/user-attachments/assets/33b6bcd8-f8bf-4785-b1af-dd115463df5f)

Purpose is to get x-chain route even before all details are known